gst-libs/gst/rtp/gstrtcpbuffer.c

```diff
@@ -109,44 +106,49 @@ gst_rtcp_buffer_validate_data_internal (guint8 * data, guint len,
   if (G_UNLIKELY (header_mask != GST_RTCP_VALID_VALUE))
     goto wrong_mask;
 
-  /* no padding when mask succeeds */
-  padding = FALSE;
-
   /* store len */
   data_len = len;
 
   while (TRUE) {
+    guint header_len;
+    gboolean padding = FALSE;
+
     /* get packet length */
     header_len = (((data[2] << 8) | data[3]) + 1) << 2;
     if (data_len < header_len)
       goto wrong_length;
 
+    /* check padding of new packet */
+    if (data[0] & 0x20) {
+      guint8 pad_bytes;
+      padding = TRUE;
+      /* last byte of padding contains the number of padded bytes including
+       * itself */
+      pad_bytes = data[data_len - 1];
```

Comment:
Probably want to reject `data_len == header_len` if the padding bit is set (otherwise, we're using the bottom byte of the length field as the padding length, which is unlikely to be a thing).

Reply:
So I wrote this one to address it: https://github.com/pexip/gst-plugins-base/pull/1/files#diff-89f869857981d183cf338bd394969e5580bce2723d9cc1273fcceba2af2b4872R1009

But it looks like it is covered, between pad_bytes being 0 (invalid) or the length of the packet being invalid.